rtos/TARGET_CORTEX/TARGET_CORTEX_M/mbed_rtx_fault_handler.h

```diff
@@ -13,27 +13,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+ 
+#ifndef MBED_RTX_FAULT_HANDLER_H
+#define MBED_RTX_FAULT_HANDLER_H 
 
 //Fault context struct
 //WARNING: DO NOT CHANGE THIS STRUCT WITHOUT MAKING CORRESPONDING CHANGES in except.S files.
 //Offset of these registers are used by fault handler in except.S
 typedef struct {
-  uint32_t R0;
-  uint32_t R1;
-  uint32_t R2;
-  uint32_t R3;
-  uint32_t R4;
-  uint32_t R5;
-  uint32_t R6;
-  uint32_t R7;
-  uint32_t R8;
-  uint32_t R9;
-  uint32_t R10;
-  uint32_t R11;
-  uint32_t R12;
-  uint32_t SP;
-  uint32_t LR;
-  uint32_t PC;
+  uint32_t R0_reg;
```

Comment:
Fault handler changes / fixes should be separate PR please

Reply:
No, refactoring exception handling to use the new error handling framework is one of the goals with this feature.